common/src/main/java/com/revenuecat/purchases/common/logUtils.kt

```diff
@@ -24,3 +24,45 @@ fun errorLog(error: PurchasesError) {
         Log.e("[Purchases] - ERROR", error.toString())
     }
 }
+
+fun debugLog(emojiList: List<String>, message: String) {
```

Comment:
I feel like the emojis list should be an implementation detail, hidden from the callers. 
And we could have multiple, specific log methods to abstract them, based on intent. 
i.e.: `debugAppleError`. 

The abstractions could even live in their own class if we don't want `logUtils` to be concerned with those specifics

Reply:
otherwise we run the risk of being inconsistent in how we use them, particularly in ordering

Reply:
Thanks for this feedback Andy! This would help with inconsistencies. I'll place them in `logUtils` to keep all the log functions together.

Reply:
Since there are emojis being placed on different log levels, would it make sense for the function to take in a param "level" and that will be the prefix of the log? For example, there are some messages with "🍎‼️" that are either on the error level ("There was a problem with the App Store.") or info level ("Could not find SKProduct for").

Reply:
I think that makes total sense